### PR TITLE
Generate feeds for categories

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -32,3 +32,8 @@ defaults:
       path: "assets/img"
     values:
       image: true
+
+feed:
+  categories:
+    - news
+    - updates
```
Comment (tripledes) [inline _config.yml]: Can't we add all the defined [categories](https://github.com/kubevirt/kubevirt.github.io/tree/master/category) ?

Comment (iranzo) [reply _config.yml]: Yes, we can, but for example 'community' category is now empty.

Also, in Pelican, you don't need to define them all to get them generated, they are just created automatically, so it's something to dig a bit more with Jekyll as long as it is our engine

Comment (iranzo) [reply _config.yml]: IMHO, categories list should be generated automatically, in the same way as 'blogs' collections, which now, are still created manually by creating a folder, a content file plus modifying the menu.

Comment (iranzo) [reply _config.yml]: What would you suggest to do? (there's a card for the community category)

Comment (tripledes) [reply _config.yml]: I'd add what we have today.

Comment (iranzo) [reply _config.yml]: Done

**Pull request merged**